lib/index.js

```diff
@@ -52,7 +52,7 @@ const listBackups = async (TableName) => {
     TimeRangeUpperBound,
   };
 
-  console.log(`Removing backups before the following date: ${moment(TimeRangeUpperBound).format('LL')}`);
+  console.log('@unly/serverless-plugin-db-backups:', `Removing backups before the following date: ${moment(TimeRangeUpperBound).format('LL')}`);
```

Comment:
Should probably use your own `log` function which does that automatically, you'll likely miss console.log this way